client/my-sites/plan-features-2023-grid/features.jsx

```diff
@@ -0,0 +1,154 @@
+
+		if ( ! feature.availableOnlyForAnnualPlans || feature.availableForCurrentPlan ) {
+			return '';
+		}
+
+		return (
+			<span className="plan-features-2023-grid__item-annual-plan">
+				{ translate( 'Included with annual plans' ) }
+			</span>
+		);
+	}
+
+	renderFeatureItem( feature, index, key = null ) {
+		const classes = classNames( 'plan-features-2023-grid__item-info', {
+			'is-annual-plan-feature': feature.availableOnlyForAnnualPlans,
+			'is-available': feature.availableForCurrentPlan,
+		} );
+
+		const itemTitleClasses = classNames( 'plan-features-2023-grid__item-title', {
+			'is-bold': feature.getSlug() === FEATURE_CUSTOM_DOMAIN,
+		} );
+		return (
+			<PlanFeaturesItem
+				annualOnlyContent={ this.renderAnnualPlansFeatureNotice( feature ) }
+				isFeatureAvailable={ feature.availableForCurrentPlan }
+			>
+				<span className={ classes } key={ key }>
+					<span className={ itemTitleClasses }>{ feature.getTitle( this.props.domainName ) }</span>
+				</span>
+			</PlanFeaturesItem>
+		);
+	}
+
+	renderEnterpriseClientLogos() {
+		return (
+			<div className="plan-features-2023-grid__item plan-features-2023-grid__enterprise-logo">
+				<img src={ TimeLogo } alt="WordPress VIP client logo for TIME" loading="lazy" />
+				<img src={ SlackLogo } alt="WordPress VIP client logo for Slack" loading="lazy" />
+				<img src={ DisneyLogo } alt="WordPress VIP client logo for Disney" loading="lazy" />
+				<img src={ CNNLogo } alt="WordPress VIP client logo for CNN" loading="lazy" />
+				<img src={ SalesforceLogo } alt="WordPress VIP client logo for Salesforce" loading="lazy" />
+				<img src={ FacebookLogo } alt="WordPress VIP client logo for Facebook" loading="lazy" />
+				<img src={ CondenastLogo } alt="WordPress VIP client logo for Conde Nast" loading="lazy" />
+				<img src={ BloombergLogo } alt="WordPress VIP client logo for Bloomberg" loading="lazy" />
+			</div>
+		);
+	}
+
+	renderPlanFeatures( features, jpFeatures, planName, mapIndex ) {
+		if ( PLAN_ENTERPRISE_GRID_WPCOM === planName ) {
+			return this.renderEnterpriseClientLogos();
+		}
+
+		const featureItemJSX = features.map( ( currentFeature, featureIndex ) => {
+			const classes = classNames( '', getPlanClass( planName ), {
+				'is-last-feature': featureIndex + 1 === features.length,
+			} );
+			const key = `${ currentFeature.getSlug() }-${ featureIndex }`;
+
+			return (
+				<div key={ key } className={ classes }>
+					{ this.renderFeatureItem( currentFeature, mapIndex, key ) }
+				</div>
+			);
+		} );
+
+		jpFeatures.length !== 0 &&
+			featureItemJSX.push(
+				<div className="plan-features-2023-grid__jp-logo" key="jp-logo">
+					<JetpackLogo size={ 16 } />
+				</div>
+			);
+
+		featureItemJSX.push(
+			jpFeatures.map( ( currentFeature, featureIndex ) => {
+				const classes = classNames( '', getPlanClass( planName ), {
+					'is-last-feature': featureIndex + 1 === features.length,
+					'is-bold': currentFeature.getSlug() === FEATURE_CUSTOM_DOMAIN,
```

Comment:
This is probably not required since custom domains are not a Jetpack feature.

Reply:
Again, fixed in a downstream PR :), check https://github.com/Automattic/wp-calypso/pull/71558/files#diff-0b686372ffee08554df93d8e28e2994f1af438f57c6e63365587efc9a5529454R20